SDWebImage/FLAnimatedImage/FLAnimatedImageView+WebCache.m

```diff
@@ -163,8 +175,12 @@ - (void)sd_setImageWithURL:(nullable NSURL *)url
                                    if (![url isEqual:sstrongSelf.sd_imageURL]) { return ; }
                                    // Step 5. Set animatedImage or normal image
                                    if (animatedImage) {
-                                       if (sstrongSelf.sd_cacheFLAnimatedImage) {
+                                       if (sstrongSelf.sd_cacheFLAnimatedImage && SDImageCache.sharedImageCache.config.shouldCacheImagesInMemory) {
```

Comment:
This is previous version bug. It does not check whether user disable memory cache or not.